templates-java/src/main/java/br/com/templates_java/config/jms_aws_sqs/SQSPropertiesGenerator.java

```diff
@@ -21,6 +21,7 @@ public File runGenerate() throws IOException {
                 "cloud.aws.region.static=us-east-1\n" +
                 "cloud.aws.stack.auto=false\n" +
                 "cloud.aws.sqs.queue-name=my-queue.fifo");
+        this.generatorOptions.setName("application.properties");
```

Comment:
Consider using a constant for the filename instead of hardcoding it, for example `private static final String CONFIG_FILE_NAME = "application.properties";` and then use it here: `this.generatorOptions.setName(CONFIG_FILE_NAME);`